api/tests/test_application_api.py

```diff
@@ -762,6 +773,17 @@ def test_unit_admin_cant_view_application(
     assert response.status_code == 404
 
 
+@pytest.mark.django_db
+def test_unit_group_admin_cant_view_application(
```

Comment:
To make this more clear, should this be renamed `test_unit_group_admin_cant_view_other_group_application`?

Now, one test says unit group admin should be able to view application and this test says unit group admin can't view application. This makes it more clear why the admin should not be able to see the application.

Reply:
changed this and other test naming a bit. added `_not_for_<unit/unitgroup)>`